### PR TITLE
Adding metadata to material page details and editions/details [DDFLSBP-543]

### PR DESCRIPTION
**Pull request by clausbruun**
#### Link to issue

https://reload.atlassian.net/browse/DDFLSBP-543

#### Description

Adding metadata to material page details and editions/details:
* Author roles
* Audience
* Notes
* Physical Description
* Extended Language info
* Source
* Parts
* Host Publication

#### Screenshot of the result

image

image

image

#### Additional info

The corresponding PR adding translation strings in `dpl-cms` is https://github.com/danskernesdigitalebibliotek/dpl-cms/pull/1050

### CODE REVIEW

File: src/apps/material/helper.ts [hunk 1]
```diff
@@ -157,7 +157,15 @@ export const getManifestationOriginalTitle = (manifestation: Manifestation) => {
 export const getManifestationContributors = (manifestation: Manifestation) => {
   return (
     manifestation.contributors
-      .map((contributor) => contributor.display)
+      .map((contributor) => {
+        if (contributor.roles.length > 0) {
+          const roleNames = contributor.roles
```
Comment (spaceo) [inline src/apps/material/helper.ts:188]: These TS errors are a bit alarming.
It seesm like you did not run the FBI codegen cmd after updating the fragment?

File: src/apps/material/helper.ts [hunk 2]
```diff
@@ -157,7 +157,15 @@ export const getManifestationOriginalTitle = (manifestation: Manifestation) => {
...
+            .map((role) => role.function.singular)
```
Comment (spaceo) [inline src/apps/material/helper.ts:189]: Same as above

**Review by spaceo — CHANGES_REQUESTED** [review]
I think you need to run codegen? The types are not matching...

File: src/apps/material/helper.ts (hunk 1 above)
Comment (clausbruun) [reply src/apps/material/helper.ts:188]: Hi @spaceo
The schema was updated in this commit: https://github.com/danskernesdigitalebibliotek/dpl-react/pull/1146/commits/0b2405e4d24ff485b1689ee2f1195555dc68b64c#diff-d7e14ab57cd19ae792d595c71bf36d4157c7fea99e0a7ea96e1003494b0baf6e 🙂

**Review by spaceo — APPROVED** [review]
LGTM 👍

**Pull request merged**